### PR TITLE
Upgrade to OCP 4.6

### PR DESCRIPTION
**Pull request by blues-man**
* Updated to OCP 4.6
* Updated to Pipelines Operator to 1.2.1
* Using @cedricclyburn content from [learn.openshift.com](https://github.com/openshift-labs/learn-katacoda/tree/master/middleware/pipelines) converting from Markdown to Asciidoc

**TODO before merge**:
- [x] Fix homeroom locally
- [x] Test with agnosticd
- [x] Test on RHPDS

### CODE REVIEW

**Comment by blues-man** [comment]
Updated agnosticd fork to point to this repo: https://github.com/blues-man/agnosticd/commit/973d86c6d7ff448212832346821f2f6fde35be79

**Comment by blues-man** [comment]
I got this error while trying to execute [this command](https://github.com/blues-man/lab-tekton-pipelines/edit/4.6/workshop/content/03create-task.adoc#L5):

```
oc project
Error from server (Forbidden): To provision Projects you must request access in https://labs.opentlc.com or https://rhpds.redhat.com.
```

do we need some fix on RHPDS/agnosticd/agnosticv side for it @thoraxe ?

**Comment by blues-man** [comment]
I also got this error while trying to use buildah task:

```
+ buildah --storage-driver=overlay push --tls-verify=false --digestfile /workspace/source/image-digest image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui docker://image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui
Getting image source signatures
unauthorized: authentication required
```

File: .workshop/build [hunk 1]
```diff
@@ -20,7 +20,7 @@ mv -f code/* . && rmdir code
 mkdir -p .bin && echo "export PATH=$HOME/.bin:$PATH" >> .bash_profile
 
 # Releases for Tekton CLI at: https://github.com/tektoncd/cli/releases
-TKN_CLI_VERSION=0.7.0
+TKN_CLI_VERSION=0.13.1
```
Comment (joellord) [inline .workshop/build:23]: Should probably use 0.15.0

Comment (joellord) [reply .workshop/build:23]: Nevermind, let's use the 0.13.1 which is the same as the one in Pipelines

**Comment by wkulhanek** [comment]
@blues-man Just set `ocp4_workload_opentlc_production_remove_self_provisioner: false` in your AgnosticV config

**Pull request merged**